app/controllers/likes_controller.rb

```diff
@@ -4,12 +4,18 @@ def update
     @next = params[:next]
 
     # add to avoid list
-    current_user.avoids.build(opposed_user_id: params[:id]).save!
+    current_user.avoids.build(like_params).save!
 
     @like_info = Like.process(params[:liked] == 'true', params[:id], current_user)
 
     respond_to do |format|
       format.js
     end
   end
+
+  private
+
+  def like_params
```

Comment:
I think you're using these strong parameters to build an `Avoid` object, not a `Like` object. 

So you might want to consider making this `avoid_params` and also making a `like_params` too in order to create a new like